Lint检查失败: chat-ops.test.ts 违反 prefer-destructuring 规则

## 测试输出错误信息

```
/Users/hs3180/clawd/disclaude/src/platforms/feishu/chat-ops.test.ts
  113:13  error  Use array destructuring  prefer-destructuring
```

## 根本原因分析

在 `src/platforms/feishu/chat-ops.test.ts` 第 113 行，代码使用了属性访问方式而不是数组解构，违反了 ESLint 的 `prefer-destructuring` 规则。

## 触发提交

- 95b9842 docs: add GitHub installation support in Quick Start (Issue #587) (#613)
- d2f0147 feat(mcp): add unified messaging MCP tool for channel-agnostic messaging (Issue #590 Phase 2) (#615)
- 5b8c19c fix(rest): implement file transfer methods for REST Channel (Issue #583) (#619)

## 修复建议

将第 113 行的属性访问改为数组解构形式。例如：
```javascript
// 之前
const item = items[0];
// 之后
const [item] = items;
```

## 相关日志

```
/Users/hs3180/clawd/disclaude/src/platforms/feishu/chat-ops.test.ts
  113:13  error  Use array destructuring  prefer-destructuring
```

--- a/src/channels/feishu-channel-mention.test.ts
+++ b/src/channels/feishu-channel-mention.test.ts
@@ -39,6 +39,7 @@ vi.mock('../config/index.js', () => ({
   Config: {
     FEISHU_APP_ID: 'test-app-id',
     FEISHU_APP_SECRET: 'test-app-secret',
+    getDebugConfig: vi.fn().mockReturnValue({}),
   },
 }));
 

--- a/src/channels/feishu-channel-passive-mode.test.ts
+++ b/src/channels/feishu-channel-passive-mode.test.ts
@@ -50,6 +50,7 @@ vi.mock('../config/index.js', () => ({
   Config: {
     FEISHU_APP_ID: 'test-app-id',
     FEISHU_APP_SECRET: 'test-app-secret',
+    getDebugConfig: vi.fn().mockReturnValue({}),
   },
 }));
 

